blocks/footer/footer.css

```diff
@@ -11,3 +12,131 @@ footer .footer {
 footer .footer p {
   margin: 0;
 }
+
+/* container for footer nav and copyright */
+footer .footer > div {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+/* flex container for footer nav */
+footer .columns > div {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  gap: 10px;
+}
+
+/* two columns for mobile */
+@media (max-width: 768px) {
+  footer .columns > div {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    gap: 10px;
+  }
+
+  footer .columns > div div {
+    width: 40%;
+  }
+}
+
+/* flex items for footer links */
+footer .columns ul {
+  list-style: none;
+  margin-left: 0;
+  padding-left: 0;
+  line-height: 24px;
+}
+
+/* footer nav links */
+footer .columns ul li {
+  margin-top: 6px;
+  padding-bottom: 6px;
+}
+
+footer .columns ul li:first-of-type {
+  padding-bottom: 6px;
+}
+
+footer .columns ul li:last-of-type {
+  margin-top: 6px;
+}
+
+footer .columns ul li a[href] {
+  text-decoration: none;
+  color: var(--mt-global-color-accent-quinary);
```

Comment:
please remove this var, the value is already available via `--mt-global-color-base-primary` - in order to prevent referencing a global, you could also add a `--mt-footer-link-color: var(--mt-global-color-base-primary)`